wasmapi/go/log.go

```diff
@@ -34,17 +34,23 @@ const (
 	TraceLevel
 )
 
+var currentLogLevel = InfoLevel
```

Comment:
How is a user of the gadget able to change it? I think this idea could work if we pass the log level on gadgetInit() or create a specific function to set this from the host. (But I don't know if it's too complicated and the previous solution was simpler)